publish-hook/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,3 @@
+org.slf4j.simpleLogger.log.com.redhat.lightblue=DEBUG
+org.slf4j.simpleLogger.log.com.redhat.lightblue.util.Error=ERROR
+org.slf4j.simpleLogger.defaultLogLevel=ERROR
```

Comment:
Dont' remove, i'll change them to error.

On Tue, Sep 22, 2015 at 9:28 AM, Dennis Crissman <notifications@github.com>
wrote:

> In publish-hook/src/test/resources/simplelogger.properties
> <https://github.com/lightblue-platform/lightblue-esb-hook/pull/33#discussion_r40100609>
> :
>
> > @@ -0,0 +1,3 @@
> > +org.slf4j.simpleLogger.log.com.redhat.lightblue=DEBUG
> > +org.slf4j.simpleLogger.log.com.redhat.lightblue.util.Error=ERROR
> > +org.slf4j.simpleLogger.defaultLogLevel=ERROR
>
> Looks like this is producing too many lines for travis. Is it worth
> removing, or at least upping the error levels?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-esb-hook/pull/33/files#r40100609>
> .
>